fuel-tx/test-helpers/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "fuel-tx-test-helpers"
```

Comment:
We might want to add this package to the workspace members set, otherwise it might end up requiring its own `target` directory or produce warnings about not being in the enclosing workspace.